packages/liveblocks-core/src/room.ts

```diff
@@ -953,6 +953,14 @@ export type Room<
    */
   getAttachmentUrl(attachmentId: string): Promise<string>;
 
+  /**
+   * Gets the unread count of inbox notifications for the current user and room.
+   *
+   * @example
+   * const count = await room.getUnreadInboxNotificationsCount();
+   */
+  getUnreadInboxNotificationsCount(): Promise<number>;
```

Comment:
I suppose this is a method we wish to support, so it is fine to mark it as experimental.